select_ppx_version.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/sh -e
 
+case $1 in
+  clean) rm -f ppx/ppx_cstruct.ml; exit 0 ;;
+  conf) (* continue *) ;;
+  *) echo "supported actions: conf, clean"; exit 1 ;;
+esac
+
 V=`ocamlc -version`
 case $V in
 4.02*) SRC=402 ;;
 4.03*) SRC=403 ;;
-*) echo Unsupported OCaml version $V ;;
+4.04*) SRC=404 ;;
```

Comment:
I considered it, but I think that it's better to be clear (and dumb) rather than try to be cunning inside a general approach that is rather duplication-friendly anyway.